### PR TITLE
Keyboard shortcuts cheat-sheet

### PR DESCRIPTION
**Pull request by PaulHax**
Pressing `?` or clicking button on setting menu pops a modal:

image

### CODE REVIEW

File: src/components/KeyboardShortcuts.vue [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<template>
+  <closeable-dialog v-model="keyboardStore.settingsOpen">
+    <v-card>
+      <v-card-title class="d-flex flex-row align-center"
+        >Keyboard Shortcuts</v-card-title
```
Comment (floryst) [inline src/components/KeyboardShortcuts.vue:5]: Another prettier styling snafu here. Not sure if this is something we can configure.

**Comment by floryst** [comment]
Looks good! Two minor comments:
- Using esc to close the popup would be nice
- Let's try a monospace font for the letters. I think that would look more consistent.

File: src/components/KeyboardShortcuts.vue (hunk 1 above)
Comment (PaulHax) [reply src/components/KeyboardShortcuts.vue]: Fixed.  🤷 I'm getting used to it.

**Comment by PaulHax** [comment]
Fixed the Escape key and added monospacing on the keybindings column.

File: src/components/KeyboardShortcuts.vue (hunk 1 above)
Comment (floryst) [reply src/components/KeyboardShortcuts.vue]: It might be fixed in a newer version of prettier, or there's some vue-prettier plugin we need.

**Pull request merged**